> Looks like crypto/ed25519 go/format net are also passing?

The net package passes for me, but the other two don't:

```
$ tinygo test crypto/ed25519
ld.lld-11: error: undefined symbol: crypto/ed25519/internal/edwards25519/field.feSquare
>>> referenced by fe.go:305 (/usr/local/go1.17/src/crypto/ed25519/internal/edwards25519/field/fe.go:305)
>>>               /tmp/tinygo3886713202/main.o:((*crypto/ed25519/internal/edwards25519/field.Element).Square)

ld.lld-11: error: undefined symbol: crypto/ed25519/internal/edwards25519/field.feMul
>>> referenced by fe.go:299 (/usr/local/go1.17/src/crypto/ed25519/internal/edwards25519/field/fe.go:299)
>>>               /tmp/tinygo3886713202/main.o:((*crypto/ed25519/internal/edwards25519/field.Element).Multiply)

ld.lld-11: error: undefined symbol: syscall.RawSyscall
>>> referenced by zsyscall_linux_amd64.go:1661 (/usr/local/go1.17/src/syscall/zsyscall_linux_amd64.go:1661)
>>>               /tmp/tinygo3886713202/main.o:(os.Pipe)
error: failed to link /tmp/tinygo3886713202/main: exit status 1
```

```
$ tinygo test go/format
panic: go/parser internal error: unbalanced scopes
FAIL	go/format	0.002s
FAIL
```

Really hoping for go/format support because it would be a nice feature for play.tinygo.org: formatting code directly in the browser without a server roundtrip.

--- a/Makefile
+++ b/Makefile
@@ -196,10 +196,8 @@ tinygo:
 test: wasi-libc
 	CGO_CPPFLAGS="$(CGO_CPPFLAGS)" CGO_CXXFLAGS="$(CGO_CXXFLAGS)" CGO_LDFLAGS="$(CGO_LDFLAGS)" $(GO) test $(GOTESTFLAGS) -timeout=20m -buildmode exe -tags byollvm ./builder ./cgo ./compileopts ./compiler ./interp ./transform .
 
-TEST_PACKAGES = \
+TEST_PACKAGES_BASE = \
 	compress/bzip2 \
-	compress/flate \
-	compress/zlib \
 	container/heap \
 	container/list \
 	container/ring \
@@ -231,7 +229,6 @@ TEST_PACKAGES = \
 	math/cmplx \
 	net/http/internal/ascii \
 	net/mail \
-	os \
 	path \
 	reflect \
 	sync \
@@ -242,11 +239,24 @@ TEST_PACKAGES = \
 	unicode/utf16 \
 	unicode/utf8 \
 
+# Standard library packages that pass tests natively
+TEST_PACKAGES = \
+	$(TEST_PACKAGES_BASE) \
+	compress/flate \
+	compress/zlib \
+	os \
+
+# Standard library packages that pass tests on wasi
+TEST_PACKAGES_WASI = \
+	$(TEST_PACKAGES_BASE)
+
 # Test known-working standard library packages.
 # TODO: parallelize, and only show failing tests (no implied -v flag).
 .PHONY: tinygo-test
 tinygo-test:
 	$(TINYGO) test $(TEST_PACKAGES)
+tinygo-test-wasi:
+	$(TINYGO) test -target wasi $(TEST_PACKAGES_WASI)
 
 .PHONY: smoketest
 smoketest:

